### PR TITLE
XW-1728 | Add tracking for DS Global Footer

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/XW-1728

//cc @Wikia/x-wing

### CODE REVIEW

**Comment by rogatty** [comment]
LGTM

File: extensions/wikia/AssetsManager/config.php [hunk 1]
```diff
@@ -2090,6 +2091,15 @@
 	)
 );
 
+/** DesignSystem extension */
+$config['design_system_js'] = array(
+	'type' => AssetsManager::TYPE_JS,
+	'skin' => [ 'oasis' ],
+	'assets' => array(
```
Comment (Grunny) [inline extensions/wikia/AssetsManager/config.php]: This uses short array syntax on the skin so they should all be consistent, and per our conventions all the arrays should use short array syntax now anyway.

**Comment by Grunny** [comment]
Minor comment, otherwise LGTM.

File: extensions/wikia/DesignSystem/scripts/DesignSystemGlobalFooterTracking.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+	/**
+	 * Parent click handler for events in the DS Global Footer
+	 * @param {object} event
+	 */
+	function clickTrackingHandler(event) {
+		var $element, label;
+
+		//Track only primary mouse button click
+		if (event.type === 'mousedown' && event.which !== 1) {
```
Comment (Warkot) [inline extensions/wikia/DesignSystem/scripts/DesignSystemGlobalFooterTracking.js:20]: What if user clicks the link with the middle mouse button to open the link in a different tab? Or right-click and then open/open in new tab? :|

Comment (kvas-damian) [inline extensions/wikia/DesignSystem/scripts/DesignSystemGlobalFooterTracking.js:20]: We don't care about them ;)

Comment (Warkot) [inline extensions/wikia/DesignSystem/scripts/DesignSystemGlobalFooterTracking.js:20]: 😮

**Pull request merged**